bbc/simorgh
# Add Naidheachdan publisher logo graphic

Resolves JIRA N/A

Overall changes
======
* Adds a new publisher graphic to public/naidheachdan, as requested by Syndication team

Code changes
======

- Adds a new publisher graphic to public/naidheachdan

Testing
======
1. Visit https://static.files.bbci.co.uk/ws/simorgh-assets/public/naidheachdan/images/metadata/publisher-nx60.png
2. See 200 and correct graphic

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
